Makefile

```diff
@@ -58,18 +58,20 @@ checkout-ethereum-package: ethereum-package ## 📦 Checkout specific Ethereum p
 		git fetch && \
 		git checkout $(ETHEREUM_PACKAGE_REVISION)
 
+ENCLAVE ?= lambdanet
+
```

Comment:
This change was made to been able to compare two different executions at the same time using `make localnet`.